src/feed/Types.hpp

```diff
@@ -19,12 +19,12 @@
 
 #pragma once
 
-#include "web/interface/ConnectionBase.hpp"
+#include "web/SubscriptionContextInterface.hpp"
 
 #include <memory>
 
 namespace feed {
-using Subscriber = web::ConnectionBase;
+using Subscriber = web::SubscriptionContextInterface;
 using SubscriberPtr = Subscriber*;
 using SubscriberSharedPtr = std::shared_ptr<Subscriber>;
```

Comment:
is it web::SubscriptionContextPtr ? Can we unify the names here? In subscribe.hpp, we are using `SubscriptionContextPtr`, in unsubscribe.hpp, we are using `SubscriberSharedPtr`.

Reply:
I changed the naming in `subscribe.cpp`.